docs/empyrical.tests.rst

```diff
@@ -0,0 +1,22 @@
+empyrical\.tests package
```

Comment:
probably don't need this either

Reply:
Will delete this file in the next commit, but again it will re-appear if make_docs.bat is run again